prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```

Comment:
Hi Gregg, the `flex-basis` property should be used on "flex items", that is, elements who are children of an element who has `display: flex`. So you want to move your `flex-basis` to the `.card-container` rule-set.

Reply:
Is there a reason when I do this that the cards are not six across? I changed the container but it still only shows 4 across in the wrap and does not look like the photo in the example.